src/library/librarycontrol.h

```diff
@@ -65,27 +79,51 @@ class LibraryControl : public QObject {
     void slotDecrementFontSize(double v);
 
   private:
-    Library* m_pLibrary;
-
-    ControlObject* m_pSelectNextTrack;
-    ControlObject* m_pSelectPrevTrack;
-    ControlObject* m_pSelectTrack;
-
-    ControlObject* m_pSelectSidebarItem;
-    ControlObject* m_pSelectPrevSidebarItem;
-    ControlObject* m_pSelectNextSidebarItem;
-
-    ControlObject* m_pToggleSidebarItem;
-    ControlObject* m_pLoadSelectedIntoFirstStopped;
-    ControlObject* m_pAutoDjAddTop;
-    ControlObject* m_pAutoDjAddBottom;
-
-    ControlObject* m_pFontSizeKnob;
-    ControlPushButton* m_pFontSizeIncrement;
-    ControlPushButton* m_pFontSizeDecrement;
-
-    WLibrary* m_pLibraryWidget;
-    WLibrarySidebar* m_pSidebarWidget;
+    Library* m_pLibrary = nullptr;
+
+    // Simulate pressing a key on the keyboard
+    void emitKeyEvent(QKeyEvent&& event);
+
+    // Controls to navigate vertically within currently focussed widget (up/down buttons)
+    ControlPushButton* m_pMoveUp = nullptr;
```

Comment:
In this case, the Initialisation here is redundant since they are initialized in the constructor again. It should be removed. This code inlines a delegating constructor, in each user code.  

----

General : 
You use here a new C++ feature, which is used in Mixxx here the first time. 
"Non-static data member initializers"

There are some pros and cons discussed for this feature: 
https://groups.google.com/a/chromium.org/forum/#!topic/chromium-dev/zqB-DySA4V0

In my real live, I us recently C# and it also supports this feature. It allows to spread class initialisation to the whole file which is from my experience not very readable.
In the current Mixxx code you just need to look at the constructor. If we allow to do it in the class definition we brake this rule. 

I am not strictly against this feature, but we should come to a conclusion how we will use it in Mixxx in future. 
For example it might be nice to use it for inline classes, or If there is would be only an empty constructor with an initialisation list. 

Any thoughts? 
@rryan @uklotzde